src/content/doc-cloud/advanced-topics/monitoring/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 4
+sidebar_label: Monitoring
+title: Monitoring | Surreal Cloud
```

Comment:
Should this be SurrealDB Cloud?

Reply:
Nah, always been Surreal Cloud in all messaging.